changelog.d/20240918_121327_maria.magallanes_docs_tutor_version.md

```diff
@@ -0,0 +1 @@
+[Documentation] Add compatibility notes, so you find info about what version of Tutor you should install by the Open edX release you need. (by @MaferMazu)
```

Comment:
This entry is unnecessary for documentation changes: https://docs.tutor.edly.io/tutor.html#contributing